google-apis-core/spec/google/apis/core/http_command_spec.rb

```diff
@@ -329,70 +329,62 @@ class DecryptResponse
     end
   end
 
-  context('with opencensus integration') do
-    it 'should create an opencensus span for a successful call' do
+  context('with OpenTelemetry integration') do
+    let(:instrumentation) { Google::Apis::Core::Instrumentation.instance }
+    let(:tracer) { instrumentation.tracer }
+
+    before { instrumentation.install }
+
+    it 'should create an OpenTelemetry span for a successful call' do
       stub_request(:get, 'https://www.googleapis.com/zoo/animals').to_return(status: [200, ''], body: "Hello world")
       command = Google::Apis::Core::HttpCommand.new(:get, 'https://www.googleapis.com/zoo/animals')
-      OpenCensus::Trace.start_request_trace do |span_context|
-        command.execute(client)
-        spans = span_context.build_contained_spans
-        expect(spans.size).to eql 1
-        span = spans.first
-        expect(span.name.value).to eql '/zoo/animals'
-        expect(span.status.code).to eql 0
-        attrs = span.attributes
-        expect(attrs['http.host'].value).to eql 'www.googleapis.com'
-        expect(attrs['http.method'].value).to eql 'GET'
-        expect(attrs['http.path'].value).to eql '/zoo/animals'
-        expect(attrs['http.status_code']).to eql 200
-        events = span.time_events
-        expect(events.size).to eql 2
-        expect(events[0].type).to eql :SENT
-        expect(events[0].uncompressed_size).to eql 0
-        expect(events[1].type).to eql :RECEIVED
-        expect(events[1].uncompressed_size).to eql 11
+      span_double = instance_double(OpenTelemetry::Trace::Span)
```

Comment:
I don't typically reach for doubles, but I was apprehensive towards including `opentelemetry-sdk` so we could use the in memory exporter to assert the generated spans.

Reply:
Why were you apprehensive? Perfectly happy including the SDK gem as a development dependency (not a runtime dependency) so it's available for tests, if it will make this cleaner.

Reply:
>Why were you apprehensive?

I just didn't want to assume that you would want the SDK included so I opted to present a quick alternative first so I could get your feedback.  I will changes this to use the SDK as I address the rest of your feedback.